src/core/Akka.Streams/Implementation/Fusing/Ops.cs

```diff
@@ -2547,7 +2547,9 @@ private sealed class Logic : InAndOutGraphStageLogic
         {
             private sealed class Holder<T>(object? message, Result<T> element)
             {
-                public object? Message { get; private set; } = message;
+                private Directive? _cachedDirective = null;
```

Comment:
Need to cache `Supervision.Directive` to avoid multiple calls to `Supervision.Decider`

Reply:
The decider isn't that expensive of a call though - why do we need to cache this?